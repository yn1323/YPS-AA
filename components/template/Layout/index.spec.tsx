import React from 'react'
import Component from '.'
import { mountWithTheme } from '@test/TestWrapper'

test(`${Component.name}-snapshot`, () => {
  const wrapper = mountWithTheme(
    <Component>
      <div>hoge</div>
    </Component>
  )
  expect(wrapper).toMatchSnapshot()
})
