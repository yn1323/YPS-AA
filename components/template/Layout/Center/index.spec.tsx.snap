// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template/Layout/Center-snapshot 1`] = `
<Center>
  <Center__Container>
    <main
      className="Center__Container-sc-tnkcpe-0 lkosJD"
    >
      <Center__CenterDiv>
        <div
          className="Center__CenterDiv-sc-tnkcpe-1 frNToq"
        />
      </Center__CenterDiv>
    </main>
  </Center__Container>
</Center>
`;
